guilford_sent_component.yaml

```diff
@@ -0,0 +1,597 @@
+          hashFunction: "sha256"
+      general.datetime:
+        type: "general.datetime"
+        column: "DATE OFFENSE COMMITTED - END"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+    name: "chargedwith1"
+
+  chargedwith2:
+    fqn: "ol.chargedwith"
+    entitySetName: "GuilfordChargedWith"
+    src: "offender"
+    dst: "charge"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: [OFFENDER NC DOC ID NUMBER, "OFFENSE QUALIFIER CODE", "SENTENCING PENALTY CLASS CODE", "NUMBER OF COUNTS", "PRIMARY OFFENSE CODE", "NC GENERAL STATUTE NUMBER", "PRIMARY FELONY/MISDEMEANOR CD."]
+      general.datetime:
+        type: "general.datetime"
+        column: "DATE OFFENSE COMMITTED - END"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+    name: "chargedwith2"
+
+  clearedby:
+    fqn: "ol.clearedby"
+    entitySetName: "GuilfordClearedBy"
+    src: "case"
+    dst: "disposition"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["COMPONENT DISPOSITION CODE", "COURT DOCKET NUMBER"]
+    name: "clearedby"
+
+  resultsin1:
+    fqn: "ol.resultsin"
+    entitySetName: "GuilfordResultsIn"
+    src: "chargeevent"
+    dst: "sentence"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["INMATE SENTENCE  STATUS CODE", "SENTENCING PENALTY CLASS CODE", "SPLIT SENTENCE ACTIVE TERM", "PRIMARY OFFENSE CODE", "NUMBER OF COUNTS", "TYPE OF COUNT CODE", "PRIMARY FELONY/MISDEMEANOR CD.", "ORIGINAL SENTENCE AUDIT CODE", "SENTENCE TYPE CODE_1", "OFFENSE QUALIFIER CODE", "SENTENCE TYPE CODE", "COMPONENT DISPOSITION CODE", "SENTENCE CONVICTION DATE", "NC GENERAL STATUTE NUMBER"]
+    name: "resultsin1"
+
+  resultsin2:
+    fqn: "ol.resultsin"
```

Comment:
could also be unneccesary

Reply:
Also let's add sentence ->results in->jailstay, there's nothing connecting them.